docs/Compounds.md

```diff
@@ -0,0 +1,12 @@
+# Compounds page
+Link: mwet.lbl.gov/compounds
```

Comment:
what is this?

Reply:
The link is meant to link to this section of the site